tests/utils_tests/test_functional.py

```diff
@@ -130,3 +131,11 @@ def test_lazy_equality(self):
 
         self.assertEqual(lazy_a(), lazy_b())
         self.assertNotEqual(lazy_b(), lazy_c())
+
+    def test_lazy_repr(self):
+        """
+        Tests that a proxy of lazy object's repr equals to the object's value
+        """
+        original_object = 'Lazy translation text'
+        lazy_obj = ugettext_lazy(original_object)
+        self.assertEquals(original_object, repr(lazy_obj))
```

Comment:
I think probably two is enough, one with a string and one with any non-string object (int is fine). Though I suppose maybe having one for bytes is worthwhile too. The bytes one doesn't look quite right to me: it's still using an int, and then has an extra cast to `bytes` in the assertion line that I don't think ought to be needed.

Also, maybe use the lambda technique for the string test, too, rather than `ugettext_lazy`?